### PR TITLE
build script to build and test for package oauth2

### PR DESCRIPTION
**Pull request by vikasgupta8**
build script to build and test for package oauth2

### CODE REVIEW

File: o/oauth2/oauth2_ubi8.4.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#
+PACKAGE_NAME=oauth2
+PACKAGE_PATH=golang.org/x/oauth2
+PACKAGE_VERSION=${1:-master}
```
Comment (seth-priya) [inline o/oauth2/oauth2_ubi8.4.sh:20]: it is probably okay to use master when there are no tags or releases but please put a comment to state that explicitly.

File: o/oauth2/oauth2_ubi8.4.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+echo "Building $PACKAGE_PATH with $PACKAGE_VERSION"
+
+if ! go get -d -u -t $PACKAGE_PATH; then
+	echo "------------------$PACKAGE_NAME:install_ failed-------------------------"
+	exit 0
+fi
+
+#/home/tester/go/pkg/mod/golang.org/x/oauth2@v0.0.0-20211104180415-d3ed0bb246c8
+cd $(ls -d $GOPATH/pkg/mod/$PACKAGE_PATH@v0.0.0-20211104180415-d3ed0bb246c8)
```
Comment (seth-priya) [inline o/oauth2/oauth2_ubi8.4.sh:43]: could this have been used as the PACKAGE_VERSION instead of "master"?

File: o/oauth2/oauth2_ubi8.4.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+echo `pwd`
+
+# Ensure go.mod file exists
+go mod init $PACKAGE_PATH
+
+go get ./...
```
Comment (seth-priya) [inline o/oauth2/oauth2_ubi8.4.sh:50]: is this step required?

**Review by seth-priya — CHANGES_REQUESTED** [review]

**Comment by seth-priya** [comment]
Some generic comments:
- please name the script as <package_name>_<distro>_<distro_version.sh for consistency
- please do not remove the content from PR template, check the appropriate checkboxes as applicable

File: o/oauth2/oauth2_ubi8.4.sh (hunk 1 above)
Comment (vikasgupta8) [reply o/oauth2/oauth2_ubi8.4.sh]: Updated version.

File: o/oauth2/oauth2_ubi8.4.sh (hunk 3 above)
Comment (vikasgupta8) [reply o/oauth2/oauth2_ubi8.4.sh]: Removed this step as it is not required here. Working fine without this step.

File: o/oauth2/oauth2_ubi8.4.sh (hunk 2 above)
Comment (vikasgupta8) [reply o/oauth2/oauth2_ubi8.4.sh:43]: Yes, Added this commit as package version.

**Review by seth-priya — APPROVED** [review]

**Pull request merged**